content/en/news/releases/1.22.x/announcing-1.22/upgrade-notes/index.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Istio 1.22 Upgrade Notes
+description: Important changes to consider when upgrading to Istio 1.22.x.
+weight: 20
+publishdate: 2024-05-13
+---
+
+When you upgrade from Istio 1.21.x to Istio 1.22.0, you need to consider the changes on this page.
+These notes detail the changes which purposefully break backwards compatibility with Istio 1.21.x.
+The notes also mention changes which preserve backwards compatibility while introducing new behavior.
+Changes are only included if the new behavior would be unexpected to a user of Istio 1.21.x.
+
+## New ambient mode waypoint attachment method
```

Comment:
I prefer to have this bullet last. Not many users would hit this, if any at all.

Also, we should link to docs that explain the attachment: https://preliminary.istio.io/latest/docs/ambient/usage/l7-features/#targeting-policies-or-routing-rules